### ISSUE TITLE
Add the support for `--all-projects` when using the Python `snyk/snyk-cli:python` container images

### ISSUE THREAD

**Issue by JPLachance** [snyk/cli#2753]
Greetings,

I've a "mono-repo" that contains multiple small Python projects. I'm trying to use the `snyk/snyk-cli:python-3.8` container image to scan all the projects of my repo in one pass.

Basically, what I want looks a lot like this:
```
docker run --rm \
  -e SNYK_TOKEN=redacted \
  -v /Users/jplachance/Projects/project:/project \
  snyk/snyk-cli:python-3.8 test \
  --all-projects
```

Now, I quickly faced an issue: `Failed to get dependencies for all 5 potential projects.`

Obviously, when running inside a container, Python dependencies are not available. We need to install the inside the container for the scan to work.

So, turns out this feature already exists: https://github.com/snyk/snyk/blob/master/docker/docker-python-entrypoint.sh#L45

By passing the `TARGET_FILE` environment variable to the container, the entry point script installs them all.

```
docker run --rm \
  -e SNYK_TOKEN=redacted \
  -e TARGET_FILE=requirements-dev.txt \
  -v /Users/jplachance/Projects/project:/project \
  snyk/snyk-cli:python-3.8 test \
  --all-projects
```

But then we face the following: `The following option combination is not currently supported: file + all-projects`

What?

Well: https://github.com/snyk/snyk/blob/master/docker/docker-python-entrypoint.sh#L38

Basically, when we use `TARGET_FILE` environment variable, it automatically creates a `TEST_SETTINGS` variable that is then used by the Snyk CLI, which conflicts with `--all-projects`.

So, how can we use `--all-projects` with the `snyk/snyk-cli:python` images?

This issue is really specific to these Python images and does not affect other images.

Thank you very much for your help!

**Comment by erichusband** [comment]
Hey JP! You can add arguments to your docker script inline like this:

```
docker run --rm \
  -e SNYK_TOKEN=redacted \
  -e TARGET_FILE=requirements-dev.txt \
  -v /Users/jplachance/Projects/project:/project \
  snyk/snyk-cli:python-3.8 test --all-projects
```

There's a bit of an issue with how you're using TARGET_FILE alongside --all-projects. Those two don't generally work together!

A recent example that I've used with some success is:

```docker run --rm -it --env SNYK_TOKEN=$SNYK_TOKEN -v $(PWD):/project snyk/snyk-cli:python-3.9 test --command=python3 --skip-unresolved```

I hope that this helps!

**Comment by JPLachance** [comment]
After discussions with the Snyk team, the feature won't be added to this image.
